src/python/pants/backend/python/dependency_inference/default_module_mapping.py

```diff
@@ -190,6 +206,9 @@ def two_groups_hyphens_two_replacements_with_suffix(
     "pysocks": ("socks",),
     "pytest": ("pytest", "_pytest"),
     "pytest-runner": ("ptr",),
+    "python-dateutil": ("dateutil",),
+    "python-dotenv": ("dotenv",),
+    "python-ipware": ("ipware",),
```

Comment:
As above, these ones may be covered by the catch-all `python-...` rule above: https://github.com/pantsbuild/pants/blob/73d41b4c9935d18a2aa94c3bff6d8f99e0cc0dcb/src/python/pants/backend/python/dependency_inference/default_module_mapping.py#L102

```suggestion
```

Reply:
Gotcha - will remove them